README.md

```diff
@@ -212,8 +212,13 @@ ESLint doesn't natively support experimental ECMAScript language features. You c
 
 Once a language feature has been adopted into the ECMAScript standard, we will accept
 issues and pull requests related to the new feature, subject to our [contributing
-guidelines](http://eslint.org/docs/developer-guide/contributing). Until then, please use
-the appropriate parser and plugin(s) for your experimental feature.
+guidelines](http://eslint.org/docs/developer-guide/contributing). Note that we will
```

Comment:
Not a fan of this wording and not sure we need this much extra text. It seems like we can just add "(stage 4 according to TV39)" into line 213 and call it a day. 

If you want to add something about stage 3, I'd do that as a separate FAQ.